_meta/config/processors.yml.tmpl

```diff
@@ -3,7 +3,8 @@
 # Configure processors to enhance or manipulate events generated by the beat.
 
 processors:
-  - add_host_metadata: ~
+# in case you run in EKS/Kubernetes environment, you can use the following processor to add node metadata
+# - add_host_metadata: ~
```

Comment:
What would be the `host.*`/`user.*` values in KSPM/EKS where this processor is not enabled?

Reply:
I have added a function which always enables host metadata for KSPM. See https://github.com/elastic/cloudbeat/pull/2126/commits/3292c21a25a2f22b9f76a810b9882f8427917b07